File: data/museum/oseg_generated/python/DeleteSpecialEvent_default_example.py
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        api_caller = api.EventsApi(api_client)

        response = api_caller.delete_special_event(
            event_id="dad4bce8-f5cb-4078-a211-995864315e39",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Events#delete_special_event: %s\n" % e)

File: data/museum/oseg_generated/csharp/DeleteSpecialEvent_default_example.cs
using System;
using System.Collections.Generic;
using System.IO;

using Org.OpenAPIMuseum.Api;
using Org.OpenAPIMuseum.Client;
using Org.OpenAPIMuseum.Model;

public class DeleteSpecialEventDefaultExample
{
    public static void Main()
    {
        var config = new Configuration();

        try
        {
            var apiCaller = new EventsApi(config);

            var response = apiCaller.DeleteSpecialEvent(
                eventId: "dad4bce8-f5cb-4078-a211-995864315e39"
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Events#DeleteSpecialEvent: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
